client/helpers/scenes.ts

```diff
@@ -251,6 +272,152 @@ export async function createSceneItems({
   return { leaves, queuedSceneItems };
 }
 
+/**
+ * Create a scene with scene items.
+ */
+export async function createSceneAndSceneItemsEXPERIMENTAL({
```

Comment:
This duplicates most of the original. Once we've proven it out, this will become the new default.